doc/building_blocks/manual_pipeline.rst

```diff
@@ -194,9 +211,16 @@ Again the visualization code is simple. We can use an fMRI slice as a
 background and plot the weights. Brighter points have a higher
 discriminating weight.
 
-.. literalinclude:: ../../examples/plot_decoding_tutorial.py
-    :start-after: # We can plot the weights, using the subject's anatomical as a background
-    :end-before: ###########################################################################
+.. code-block:: python
+
+    from nilearn.plotting import plot_stat_map, show
+
+    plot_stat_map(coef_img, bg_img=haxby_dataset.anat[0],
+                  title="SVM weights", display_mode="yx")
```

Comment:
Maybe we could use a "view_stats_map" here: it now worked in the sphinx-gallery reports.

Reply:
`view_stats_map` is interactive. Is it the right thing to do in a pipeline (I'm going solely by the title of the RST file)? 
We can't save the view_image output in an automated workflow. (Or can we? It is a JS HTML Page... hmmm a PR for that?)